TrackerConditions/src/StrawElectronics.cc

```diff
@@ -70,18 +70,14 @@ namespace mu2e {
       response[i] *= 1 / gain_160;
   }
 
-  static std::vector<double> strawLengths = {1188.71,1184.67,1180.58,1176.44,1172.25,1168.01,1163.73,1159.39,1155.01,1150.57,1146.08,1141.55,1136.95,1132.31,1127.61,1122.86,1118.05,1113.19,1108.27,1103.29,1098.26,1093.16,1088.01,1082.79,1077.52,1072.18,1066.77,1061.31,1055.78,1050.18,1044.51,1038.78,1032.98,1027.1,1021.16,1015.14,1009.04,1002.87,996.624,990.297,983.89,977.401,970.829,964.171,957.426,950.592,943.668,936.65,929.537,922.328,915.018,907.607,900.092,892.469,884.737,876.891,868.93,860.85,852.648,844.32,835.862,827.27,818.541,809.669,800.65,791.479,782.15,772.658,762.997,753.161,743.141,732.931,722.523,711.908,701.077,690.019,678.723,667.178,655.369,643.283,630.904,618.214,605.193,591.82,578.07,563.916,549.326,534.264,518.691,502.557,485.807,468.376,450.183,431.133,411.107,389.953};
```

Comment:
Are these numbers extracted from TrackerGeom?

Reply:
This is only in an intermediate commit, in the final pull request it comes from TrackerGeom